web/themes/custom/cj_material/sass/styles.scss

```diff
@@ -22,11 +22,24 @@ h1, .h1 {
   text-transform: uppercase;
   margin-top: 0;
   padding-bottom: 10px;
-  font-size: 40px;
-  padding-bottom: 34px;
+  font-size: 28px;
+  line-height: 1em;
+  padding-bottom: 30px;
   position: relative;
 }
 
+@media (min-width: $xs) {
+  h1, .h1 {
+    font-size: 40px;
+  }
+}
+
+@media (max-width: $xs) {
```

Comment:
Lets change this to the mobile first pattern. i.e. all h2s are 18px unless the media has a min width of xs